cfgov/v1/util/filterable_list.py

```diff
@@ -116,3 +115,9 @@ def form_id(self):
             return form_ids[0]
         else:
             return 0
+
+    def serve(self, request):
```

Comment:
Hmmm, I was matching the signature in the example here -- http://docs.wagtail.io/en/v1.9/reference/pages/model_recipes.html